facilities/exhibition_hall_one.html

```diff
@@ -0,0 +1,385 @@
+---
+layout: default
+---
+
+<style type="text/css">
+  .hover-image {
+    position: relative;
+    top: -280px;
+    left: 0;
+    opacity: 0;
+  }
+  .hover-image:hover {
+    opacity: 1;
+  }
+  .cf {
+    float: left;
+    overflow: auto;
+    width: 100%;
+  }
+
+  .capability {
+    display: inline-block;
+    float: left;
+    height: 80px;
+    margin-right: 10px;
+    padding-top: 1em;
+    width: 80px;
+  }
+    .wifi {
+      background: url({{site.baseurl}}/assets/images/icons_sprite.png) 0 0 no-repeat;
+    }
+    .data {
+      background: url({{site.baseurl}}/assets/images/icons_sprite.png) 0 -80px no-repeat;
+    }
+    .signage {
+      background: url({{site.baseurl}}/assets/images/icons_sprite.png) 0 -160px no-repeat;
+    }
+    .voice {
+      background: url({{site.baseurl}}/assets/images/icons_sprite.png) 0 -240px no-repeat;
+    }
+    .a-v {
+      background: url({{site.baseurl}}/assets/images/icons_sprite.png) 0 -320px no-repeat;
+    }
+    .catering {
+      background: url({{site.baseurl}}/assets/images/icons_sprite.png) 0 -400px no-repeat;
+    }
+
+  .capabilities {
+    margin-left: 90px;
+  }
+  .Capabilities li {
+    line-height: 1;
+  }
+</style>
+<div class="usa-grid">
+  <h1>Level 1 - Exhibition Hall 1</h1>
+</div>
+<div class="usa-grid" style="position:relative;">
+  <div class="usa-width-two-thirds">
+    
+    <img src="{{site.baseurl}}/assets/floorplans/exhibition-hall-1.png" alt="Exhibition Hall 1">
```

Comment:
@josedigital @mattlangan Minor point, but we should probably put the these images in the /assets/images directory. Also, floor plan is two words, so could we name the subfolder `floor-plans`?